int/api/myplugin/plugin.go

```diff
@@ -0,0 +1,22 @@
+package myplugin
+
+import (
+	"github.com/massalabs/thyra/api/swagger/server/restapi/operations"
+	"github.com/massalabs/thyra/pkg/plugin"
+)
+
+func InitializePluginAPI(api *operations.ThyraServerAPI) {
+	manager := plugin.NewManager()
+	api.PluginManagerInstallHandler = newInstall(manager)
+	// api.PluginManagerExecuteCommandHandler
```

Comment:
Yes, because this endpoint is defined at the specification level, but is not implemented. So if you look at the swagger file and search for it, you'll find it commented out.

@SlnPons By the way, this means that no command handlers are implemented here. So nothing to `stop`, `kill` or `restart` a plugin process at the moment. Is this a big problem?

Reply:
@gregLibert fine by me. We'll implement these functionalities later as long as the end-point exists, the job for this ticket is done.